test/gtest/gtest_common.hpp

```diff
@@ -114,6 +114,28 @@ bool IsTestSupportedForDevMask()
     return res;
 }
 
+inline Gpu GetDevGpuType()
```

Comment:
The idea of existing `IsTestSupportedForDevMask` it to mimic behavior of CTest (it supports the same defaults). You can convert existing `*_DISABLED` and `*_ENABLED` from there to `enabled<Gpu::...>` and `disabled<Gpu::...>`, and use the latter with `IsTestSupportedForDevMask`. Why this approach can't be used in your code?